diff -urN brbox.orig/sources/CMakeLists.txt brbox/sources/CMakeLists.txt
--- brbox.orig/sources/CMakeLists.txt	2022-10-17 09:16:48.592597033 +0200
+++ brbox/sources/CMakeLists.txt	2022-10-17 09:24:00.674087111 +0200
@@ -7,7 +7,7 @@
 endif()
 add_subdirectory(lib)
 add_subdirectory(services)
-add_subdirectory(utils)
+#add_subdirectory(utils)
 #add_subdirectory(kmods)
 
 #for cross compiling
diff -urN brbox.orig/sources/lib/CMakeLists.txt brbox/sources/lib/CMakeLists.txt
--- brbox.orig/sources/lib/CMakeLists.txt	2022-10-17 09:16:48.604597073 +0200
+++ brbox/sources/lib/CMakeLists.txt	2022-10-17 09:23:46.402036513 +0200
@@ -12,7 +12,7 @@
 #    CONFIGURE_COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/libstrophe/configure --prefix=${CMAKE_INSTALL_PREFIX}
 #    BUILD_COMMAND ${MAKE})
 
-add_subdirectory(lib-onion)
+#add_subdirectory(lib-onion)
 #add_subdirectory(lib-ilclient)
 #for cross compiling
 #cmake -H. -BOutput -DCMAKE_TOOLCHAIN_FILE=toolchain-arm-backup.cmake
diff -urN brbox.orig/sources/services/CMakeLists.txt brbox/sources/services/CMakeLists.txt
--- brbox.orig/sources/services/CMakeLists.txt	2022-10-17 09:20:11.877285771 +0200
+++ brbox/sources/services/CMakeLists.txt	2022-10-17 09:24:38.202220474 +0200
@@ -5,8 +5,8 @@
 if(${MINIMUM_BRBOX_SYSTEM})
 add_subdirectory(sysmgr)
 add_subdirectory(dispsrv)
-add_subdirectory(xmproxy)
-#add_subdirectory(modbussrv)
+#add_subdirectory(xmproxy)
+add_subdirectory(modbussrv)
 else()
 add_subdirectory(sysmgr)
 #add_subdirectory(smarteye)
